docs/community/get_involved.md

```diff
@@ -7,28 +9,70 @@
 
 </div>
 
+## GTFS Governance
+
+The GTFS Specification is dynamic, evolving through an open process driven by the community. We actively participate in proposing amendments and extensions to the specification.
+
+To formalize this process, we've established the GTFS Governance framework, which includes the Specification Amendment Process (SAP).
+
+<div class="grid cards" markdown>
+
+- :material-newspaper-variant:{ .lg .middle } __GTFS Schedule (SAP)__
```

Comment:
I feel like it would be better to use the full name ("Specification Amendment Process") in the boxes, just because the boxes draw your attention more than the previous text which explains the acronym.

Reply:
Sure but the main deiferentiator is the specification rather than the SAP, lets see how it formats